docs/zh_cn/hadoop_java_sdk.md

```diff
@@ -1,82 +1,93 @@
-# 在 Hadoop 环境使用 JuiceFS Java SDK
-
-* [Haddop 兼容性](#hadoop-兼容性)
-* [编译](#编译)
```

Comment:
Why delete this?

Reply:
I think the structure after the rearrangement is clear enough, this table of contents may affect the reading experience. Still keep it?

Reply:
cc @Suave